x-pack/test/fleet_api_integration/apis/epm/final_pipeline.ts

```diff
@@ -157,9 +156,7 @@ export default function (providerContext: FtrProviderContext) {
         body: {
           message: 'message-test-1',
           event: {
-            original: {
-              foo: 'bar',
-            },
+            original: JSON.stringify({ foo: 'bar' }),
```

Comment:
The ES client seems to no longer coerce this value to a string under the hood, and `event.original` is mapped as `keyword`, so it needs to be a string.